checkov/cloudformation/checks/resource/aws/ALBListenerHTTPS.py

```diff
@@ -28,7 +28,9 @@ def scan_resource_conf(self, conf):
                         and
                         conf['Properties']['DefaultActions'][0]['Type'] == 'redirect'
                         and
-                        conf['Properties']['DefaultActions'][0]['RedirectConfig']['Protocol'] == "HTTPS"
+                        conf['Properties']['DefaultActions'][0].get('RedirectConfig')
+                        and
+                        conf['Properties']['DefaultActions'][0]['RedirectConfig'].get('Protocol') == "HTTPS"
```

Comment:
The two new conditions can be merged to
```suggestion
                        conf['Properties']['DefaultActions'][0].get('RedirectConfig', {}).get('Protocol') == "HTTPS"
```